pg_sparse/Cargo.lock

```diff
@@ -0,0 +1,2517 @@
+# This file is automatically @generated by Cargo.
```

Comment:
This file shouldn't exist, we should only have a Cargo.lock in the top-level, if you remove it, now that it's added to the top-level Cargo.toml, it shouldn't regenerate